.github/workflows/sync-upstream.yml

```
@@ -0,0 +1,146 @@
+          # Push the branch
+          git push origin $BRANCH_NAME
+
+          # Get the diff summary
+          DIFF_SUMMARY=$(git diff --stat upstream/main...HEAD || echo "Unable to generate diff")
+
+          # Create pull request
+          gh pr create \
+            --title "Sync with upstream: $(date +%Y-%m-%d)" \
+            --body "This PR automatically syncs with upstream/main and runs MCP tests.
+
+## Summary
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)